Makefile

```diff
@@ -18,6 +18,10 @@ verify-gofmt:
 test:
 	go test -v --race -cpu=1,2,4 ./code/ ./runtime/
 
+.PHONY: integration-test
+integration-test: gofail
+	./integration/sleep/execute.sh
```

Comment:
You can add the following targets:
- integration-test
  - integration-test-sleep
  - ....
- build-integration-test
  - build-integration-test-sleep
  - ...
- cleanup-integration-test
  - cleanup-integration-test-sleep
  - ...